internal/aws/containerinsightcommon/const.go

```diff
@@ -5,10 +5,11 @@ const (
 	InstanceType       = "InstanceType"
 	GoPSUtilProcDirEnv = "HOST_PROC"
 
-	MinTimeDiff    = 50 * 1000 // We assume 50 micro-seconds is the minimal gap between two collected data sample to be valid to calculate delta
-	ClusterNameKey = "ClusterName"
-	NodeNameKey    = "NodeName"
-	Version        = "Version"
+	MinTimeDiff             = 50 * 1000 // We assume 50 micro-seconds is the minimal gap between two collected data sample to be valid to calculate delta
```

Comment:
Can we use unit from time package like `50 * time.Second`

Reply:
Sure.